src/pages/oferta.tsx

```diff
@@ -32,8 +36,12 @@ const OffertPage = () => {
           firstPartAdditionalParagraph="Zobacz, co konkretnie zrobimy, aby Twój biznes się "
           secPartAdditionalParagraph="rozkręcił!"
         />
-        <BoxesOffert />
-        <OffertComponent />
+        {/* <BoxesOffert /> */}
```

Comment:
Why commented out?

Reply:
BoxesOffert is outdated but this will be section 3, I haven't started doing it yet.